shop/models/notification.py

```diff
@@ -129,7 +129,7 @@ def order_event_notification(sender, instance=None, target=None, **kwargs):
     from shop.models.order import OrderModel
     from shop.serializers.order import OrderDetailSerializer
 
-    if not isinstance(instance, OrderModel):
+    if not isinstance(instance, OrderModel._materialized_model):
```

Comment:
Yup. As long as we officially require that the order model is subclassed from `BaseOrder` (which we do per the current docs), `isinstance(instance, BaseOrder)` appears fine to me.

In general, I consider `isinstance()` a code smell. My idea for the "proper" way to fix this problem is to use a dedicated signal instead of relying on `post_transition`, that's why I didn't file the PR earlier. But since we have users running into this bug right now, fixing it quickly appears more important than fixing it properly :) We can still develop the "proper" fix later.